cmd/agent/api/v1/clcrunner.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// Package v1 implements the api endpoints for the `/api/v1` prefix.
+// This group of endpoints is meant to provide external queries with
+// stats of the agent.
+package v1
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/api/agent"
+	"github.com/DataDog/datadog-agent/pkg/status"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/gorilla/mux"
+)
+
+// SetupHandlers adds the specific handlers for /api/v1 endpoints
+func SetupHandlers(r *mux.Router) {
+	r.HandleFunc("/clcrunner/version", agent.GetVersion).Methods("GET")
+	r.HandleFunc("/clcrunner/stats", getCLCRunnerStats).Methods("GET")
+}
+
+// getCLCRunnerStats retrieves Cluster Level Check runners stats
+func getCLCRunnerStats(w http.ResponseWriter, r *http.Request) {
+	log.Info("Got a request for the runner stats. Making stats.")
```

Comment:
Might be verbose, maybe we should switch to `Debug` ?

Reply:
I wanted it to be coherent with the way we log in the private API, also the DCA is expected to query this endpoint every 10 minutes.